File: FigurasGeometricas/EntidadesCs/Cubo.cs
namespace EntidadesCs
{
	public class Cubo : Figura, Figura3D
	{
		public float Lado { get; set; }
		public Cubo(string nombre, float lado) : base(nombre)
		{
			Lado = lado;
		}
		public override string ToString()
		{
			return $"Soy {Nombre} y tengo mis lados de {Lado}";
		}

		public float CalcularVolumen()
		{
			return (float)Math.Pow(Lado, 3);
		}
	}
}

File: FigurasGeometricas/EntidadesVb/Cubo.vb
Public Class Cubo
	Inherits Figura
	Implements Figura3D

	Public Property Lado As Single
	Public Sub New(nombre As String, lado As Single)
		MyBase.New(nombre)
		Me.Lado = lado
	End Sub

	Public Function CalcularVolumen() As Single Implements Figura3D.CalcularVolumen
		Return Math.Pow(Lado, 3)
	End Function
	Public Overrides Function ToString() As String
		Return $"Soy {Nombre} y tengo mis lados de {Lado}"
	End Function
End Class
